### DISCUSSION TITLE
Unauthorized operations?

### DISCUSSION THREAD

**Discussion by AdminModTest** [Zerg00s/FlowPowerAppsMigrator#24] [Q&A]
I try to move the list on the new Sharepoint site, but i encounter the error that says i am attempted to perform an unauthorized operations Both account have sharepoint administrator role, any ideas on how to solve this? 

![image](https://user-images.githubusercontent.com/107675894/181174009-70c1d1f3-ce9f-4214-9901-5a511d3d0347.png)

**Comment by Zerg00s** [comment]
Hi @AdminModTest, chances are the account you are using is not a site collection administrator on the source site. Can you confirm, please?
